modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/ContextProviderUtil.java

```diff
@@ -113,6 +116,20 @@ public static MultivaluedHashMap<String, String> getMultivaluedHashMap(
 		};
 	}
 
+	private static Object _fetchExistingResource(
+		Exchange exchange, String... messageKeys) {
```

Comment:
I see that receiving here the `messageKeys` useless as the method is not reused anywhere else in the code, so, why just using the keys inside the method?